aloha-h2o/src/test/scala/com/eharmony/aloha/models/h2o/H2oModelTest.scala

```diff
@@ -94,29 +94,80 @@ class H2oModelTest extends Logging {
     assertFalse(rowData.contains("len_none_no"))
   }
 
+  @Test def testVectorOnEmptyData(): Unit = {
+    val model = AbaloneVecModel
 
+    val empty = Abalone.getDefaultInstance
 
-  @Test def testProto(): Unit = {
-    val model = AbaloneModel
+    val expMissing = Set(
+      "diameter",
+      "height",
+      "length",
+      "sex",
+      "weight.shell",
+      "weight.shucked",
+      "weight.viscera",
+      "weight.whole"
+    )
 
-    // For expository purposes:
-    val input:    Abalone       = AbaloneData.toStream.head
-    val expected: Double        = ExpectedAbaloneModelResults.toStream.head
-    val actual:   Option[Float] = model(input).value
-    assertEquals(expected, actual.get, Epsilon)
+    val expEmptyOut =
+      RootedTreeImpl(
+        modelId = ModelId(0, "proto model"),
+        errorMsgs = Vector(
+          "H2o model may have encountered a missing categorical variable.  Likely features: Sex",
+          "See: vec_glm_afa04e31_17ad_4ca6_9bd1_8ab80005ce38.score0(vec_glm_afa04e31_17ad_4ca6_9bd1_8ab80005ce38.java:59)"
+        ),
+        missingVarNames = expMissing,
+        value = None,
+        subvalues = Vector.empty,
+        prob = None
+      )
+
+    val emptyOut = model(empty)
+    assertEquals(expEmptyOut, emptyOut)
+  }
+
+  @Test def testVector(): Unit = {
```

Comment:
This test and the `testProto` test produce the same scores.

Reply:
This is a great idea!